tests/cli/test_buildspec.py

```diff
@@ -150,6 +152,16 @@ def test_buildspec_find_invalid():
     cache.print_invalid_buildspecs(error=False, terse=True, header=True)
 
 
+@pytest.mark.cli
+def test_edit_test():
+    edit_buildspec_test(configuration, buildspecs=[], editor="", test=True)
```

Comment:
You can try calling a valid buildspec you can do something like `buildspecs = [os.path.join(BUILDTEST_ROOT,"tutorials","vars.yml")]`